projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -27,6 +27,115 @@ function grunion_contact_form_require_endpoint() {
 	require_once GRUNION_PLUGIN_DIR . 'class-grunion-contact-form-endpoint.php';
 }
 
+/**
+ * Sets the 'block_template' attribute on all instances of wp:jetpack/contact-form in
```

Comment:
Would it make sense to add a comment to each block to say which way of adding a form it fixes clearly?

Reply:
> Would it make sense to add a comment to each block to say which way of adding a form it fixes clearly?

I think so! Great suggestion 😄 Added with 0434b8c6f01a412445779e86aeef0422c2f83f0b